src/pages/docs/tutorials/tutorial-kitten-browser/data-and-state/index.md

```diff
@@ -219,14 +239,14 @@ const App = Changeable({prop: 'index', change: 'onNavigate'},
 
 If everything has gone smoothly, you should now have a working Enact Kitten Browser with state managed by the `Changeable` HOCs flowing downstream via props and user actions flowing back upstream via events. This style of architecture will be useful as you build larger, more complex apps allowing you to decouple state and behavior from your components and views.
 
-Below is the complete source for each of files modified in this tutorial which may be useful to see how the changes introduced above should be integrated together.
+Below is the complete source for each of files modified in this tutorial which may be useful to see how the changes introduced above should be integrated together. You can get this source from our repository [Kitten Browser](https://github.com/enactjs/samples/tree/develop/tutorial-kitten-browser).
```

Comment:
You will use `develop` branch? not `master`?

Reply:
The link is removed.